### ISSUE TITLE
Drawing 2d p5 sketch on VR screen

### ISSUE THREAD

**Issue by vedhant** [stalgiag/p5.xr#51]
The most suitable method for adding 2D sketch on a 3D sketch is using `p5.Graphics()`. 

I am thinking the similar can be done for VR too. As `image()` is been called, the offset for eyes should be working as even images undergoes uMVMatrix transformation. However, it does not stick to the screen of the viewer. It has a fixed position in 3D space. To make it fix to viewer screen, I will have to take care of the uMVMatrix transformation.

The following example depicts it :
```javascript
let pg;
let randomx=[], randomy=[];

function preload() {
  createVRCanvas();
}

function setup() {
  setVRBackgroundColor(200, 0, 150);
  pg = createGraphics(windowWidth, windowHeight);
  for(let i=0; i<5; ++i) {
    randomx[i] = random(-500, 500);
    randomy[i] = random(-500, 500);
  }
}

function draw() {
  setViewerPosition(0, 0, 400);
  for(let i=0; i<5; ++i) {
    push();
    translate(randomx[i], randomy[i]);
    fill('red');
    if(intersectsSphere(70, 0, 0)) {
      fill('blue');
    }
    sphere(70);
    noLoop();
    pop();
  }
  pg.circle(windowWidth/2, windowHeight/2, 50);
  image(pg, -windowWidth/2, -windowHeight/2, windowWidth, windowHeight);
}

```

**Comment by stalgiag** [comment]
This might require a different method such as `stickyImage()`. I personally think it would be best if it is a state like `fill()` and `noFill()` and anything that you draw between `sticky()` and `noSticky()` is modified with the uMVMatrix transformation of the headset.

**Comment by vedhant** [comment]
In #53 anything can be made stickable (even 3d objects). Its like calling `push()` and `pop()`. This will also help users to drag objects very easily. Although dropping could be an issue.
The following snippet illustrates it :
```javascript
let pg;
let randomx, randomy;
let dragging = false;

function preload() {
  createVRCanvas();
}

function setup() {
  setVRBackgroundColor(200, 0, 150);
  randomx = random(-500, 500);
  randomy = random(-500, 500);
}

function draw() {
  setViewerPosition(0, 0, 400);
  push();
  translate(randomx, randomy);
  if(dragging || intersectsSphere(70, 0, 0)) {
    dragging = true;
    sticky();
    fill('blue');
    sphere(70);
    noSticky();
  }
  else {
    fill('red');
    sphere(70);
  }
  pop();
}

```

**Comment by vedhant** [comment]
If user wants to render sticky 2d sketch in vr, `p5.Graphics` must be used which will be rendered using `image()`. However if a 3d content floats above this image, this image will become hidden by the object (reticle can be blocked by a sphere). So, such a way to provide 2d sticking is not ideal.

Therefore, I think there should be a p5.xr api to provide such sticky 2d rendering.

**Comment by vedhant** [comment]
I have added functionality where the code rendered within `sticky()` will be rendered on top without being hidden by any objects in the scene.

**Comment by vedhant** [comment]
Closed with #52
